website/views/static/plans.jade

```diff
@@ -15,15 +15,15 @@ block content
       p
         =env.t('indivPlan1')
         |&nbsp;
-        a(href='http://habitica.wikia.com/wiki/Guilds' target='_blank')=env.t('guilds')
+        !=env.t('guildsLink')
         = ' ' + env.t('and') + ' '
-        a(href='http://habitica.wikia.com/wiki/Challenges' target='_blank')=env.t('challenges')
+        !env.t('challengesLink')
```

Comment:
This is missing an `=` after the `!`, causing the text to appear like this:
`free (or cheap) Guilds and !env.t('challengesLink')  can be used`